cylc/flow/wallclock.py

```diff
@@ -37,21 +37,16 @@
 TIME_FORMAT_EXTENDED = "%H:%M:%S"
 TIME_FORMAT_EXTENDED_SUB_SECOND = "%H:%M:%S.%f"
 
-TIME_ZONE_STRING_LOCAL_BASIC = get_local_time_zone_format(
-    TimeZoneFormatMode.reduced)
-TIME_ZONE_STRING_LOCAL_EXTENDED = get_local_time_zone_format(
-    TimeZoneFormatMode.extended)
 TIME_ZONE_STRING_UTC = "Z"
 TIME_ZONE_UTC_UTC_OFFSET = (0, 0)
-TIME_ZONE_LOCAL_UTC_OFFSET = get_local_time_zone()
-TIME_ZONE_LOCAL_UTC_OFFSET_HOURS = TIME_ZONE_LOCAL_UTC_OFFSET[0]
-TIME_ZONE_LOCAL_UTC_OFFSET_MINUTES = TIME_ZONE_LOCAL_UTC_OFFSET[1]
 
 TIME_ZONE_LOCAL_INFO = {
```

Comment:
This is only used in `cylc.flow.data_store_mgr.DataStoreMgr.generate_definition_elements` which seems to be for fields which are not updated. @dwsutherland might wish to comment on whether this safe.

Reply:
Well the time zone is recorded on the workflow object:
https://github.com/cylc/cylc-flow/blob/d5ac2e2b7bf349458a8723f1235013b1b301873e/cylc/flow/data_store_mgr.py#L697

So perhaps that'll need a delta or made a field that gets checked whenever other workflow deltas are created.

The job nodes just get passed the time by the events or job manager.

Reply:
That leads to a real possibility that the time zones in the GUI will be wrong on DST change?

Reply:
What is the use case for having in the GraphQL API the local system time zone on scheduler start (except not when the workflow is running in UTC mode)? Note this is not the [cycle point time zone](https://cylc.github.io/cylc-doc/latest/html/reference/config/workflow.html#flow.cylc[scheduler]cycle%20point%20time%20zone)